openupgrade_scripts/scripts/hr_attendance/17.0.2.0/pre-migration.py

```diff
@@ -12,6 +13,38 @@
         "hr_attendance.group_hr_attendance_own_reader",
     ),
 ]
+_fields_renames = [
```

Comment:
Put this inside the method, as this is not something general, and having it here can be misleading.